float32 linear_velocity
float32 yaw_rate